metricbeat/module/mongodb/status/data.go

```diff
@@ -0,0 +1,241 @@
+package status
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+)
+
+func eventMapping(status map[string]interface{}) common.MapStr {
+	errs := map[string]error{}
+	event := common.MapStr{
+		"version": mustBeString("version", status, errs),
+		"uptime": common.MapStr{
+			"ms": mustBeInteger("uptimeMillis", status, errs),
```

Comment:
Interesting approach with the errs. So far I used to helper.ToInt( functions etc.